### PR TITLE
 categories recipe

### PR DESCRIPTION
**Pull request by tuanbpl-0873**
https://edu-redmine.sun-asterisk.vn/issues/11965

### CODE REVIEW

**Comment by tuanbpl-0873** [comment]
ready

File: public/js/frontend/master.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+$(document).ready( function() {
+
+    var toastrSessionData = $('.toastr-session').data('session')
+    var toastrSessionArray = toastrSessionData.split(",")
+    //toastr js notification
+    if (toastrSessionArray[0] == 1)
+    {
+        var type = toastrSessionArray[1]
+
+        switch(type){
```
Comment (chinn01-0830) [inline public/js/frontend/master.js]: convention :v

File: public/js/frontend/master.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+$(document).ready( function() {
+
+    var toastrSessionData = $('.toastr-session').data('session')
+    var toastrSessionArray = toastrSessionData.split(",")
+    //toastr js notification
+    if (toastrSessionArray[0] == 1)
+    {
```
Comment (chinn01-0830) [inline public/js/frontend/master.js]: convention :v

File: resources/views/frontend/create-recipe/categories.blade.php [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+<link rel="stylesheet" href="{{ asset('bower_components/bootstrap/dist/css/bootstrap.min.css') }}">
+@endsection
+
+@section('content')
+
+<div class="wrap clearfix">
+    <!--breadcrumbs-->
+    <nav role="navigation" class="breadcrumbs">
+        <ul>
+            <li><a href="{{ route('home') }}" title="Home">'{{ __('Home') }}</a></li>
```
Comment (chinn01-0830) [inline resources/views/frontend/create-recipe/categories.blade.php]: i18n :v

**Pull request merged**